updatecli/updatecli.d/nginx.yaml

```diff
@@ -52,7 +52,17 @@ targets:
         keyword: "FROM"
         matcher: "nginx"
     scmid: default
-
+  updateTestHarness:
+    name: "Update the expectedError version in the test harness"
+    sourceid: latestStableRelease
+    scmid: default
+    kind: file
+    spec:
+      file: "cst.yml"
+      matchpattern: >
+        expectedError: \["(.*)"]
+      replacepattern: >
+        expectedError: ["{{ source "latestStableRelease" }}"]
```

Comment:
Is there a reason for not using the Yaml parser?

Reply:
sorry, I forgot to switch to draft, I am still working on the major.minor version without the patch and it imply using the kind yaml